x-pack/plugins/ml/public/application/components/import_export_jobs/import_jobs_flyout/import_jobs_flyout.tsx

```diff
@@ -0,0 +1,470 @@
+    },
+    [jobIds]
+  );
+
+  const DeleteJobButton: FC<{ index: number }> = ({ index }) => (
+    <EuiButtonIcon
+      iconType="trash"
+      aria-label="Delete"
+      color={deleteDisabled ? 'subdued' : 'danger'}
+      disabled={deleteDisabled}
+      onClick={() => deleteJob(index)}
+    />
+  );
+
+  return (
+    <>
+      <FlyoutButton onClick={toggleFlyout} isDisabled={isDisabled} />
+
+      {showFlyout === true && isDisabled === false && (
+        <EuiFlyout onClose={() => setShowFlyout(false)} hideCloseButton size="m">
+          <EuiFlyoutHeader hasBorder>
+            <EuiTitle size="m">
+              <h2>
+                <FormattedMessage
+                  id="xpack.ml.importExport.importFlyout.flyoutHeader"
+                  defaultMessage="Import jobs"
+                />
+              </h2>
+            </EuiTitle>
+          </EuiFlyoutHeader>
+          <EuiFlyoutBody>
+            <>
+              <div style={{ textAlign: 'center' }}>
+                <EuiFilePicker
+                  disabled={importing}
+                  fullWidth={true}
+                  id="filePicker"
+                  initialPromptText={i18n.translate(
+                    'xpack.ml.importExport.importFlyout.fileSelect',
+                    {
+                      defaultMessage: 'Select or drag and drop a file',
+                    }
+                  )}
+                  onChange={(files) => onFilePickerChange(files)}
+                  className="file-datavisualizer-file-picker"
+                />
+              </div>
+
+              {showFileReadError && <CannotReadFileCallout />}
+
+              {totalJobsRead > 0 && jobType !== null && (
+                <>
+                  <EuiSpacer size="l" />
+                  {jobType === 'anomaly-detector' && (
+                    <FormattedMessage
+                      id="xpack.ml.importExport.importFlyout.selectedFiles.ad"
+                      defaultMessage="{num} anomaly detection {num, plural, one {job} other {jobs}} read from file"
+                      values={{ num: totalJobsRead }}
+                    />
+                  )}
+
+                  {jobType === 'data-frame-analytics' && (
+                    <FormattedMessage
+                      id="xpack.ml.importExport.importFlyout.selectedFiles.dfa"
+                      defaultMessage="{num} data frame analytics {num, plural, one {job} other {jobs}} read from file"
+                      values={{ num: totalJobsRead }}
+                    />
+                  )}
+
+                  <EuiSpacer size="m" />
+
+                  <CannotImportJobsCallout
+                    jobs={skippedJobs}
+                    autoExpand={jobIds.length === 0 || skippedJobs.length === 1}
+                  />
+
+                  <FormattedMessage
+                    id="xpack.ml.importExport.importFlyout.importableFiles"
+                    defaultMessage="{num} importable {num, plural, one {job} other {jobs}}"
```

Comment:
Is `importable` a word? Maybe better replaced with a variation of `can be imported`.

Reply:
I've changed this to "Import {num} jobs" in this commit https://github.com/elastic/kibana/pull/101037/commits/04c01e9a1778555b0c7480ed2ec46c1f5567a2da